libs/agno/agno/app/copilotkit/app.py

```diff
@@ -0,0 +1,58 @@
+"""CopilotKit FastAPI application helper.
+
+Usage:
+```
+from agno.app.copilotkit.app import CopilotKitApp
+from agno.agent.builtins import EchoAgent
+
+agent = EchoAgent()
+app = CopilotKitApp(agent=agent).get_app()
+```
+"""
+
+from __future__ import annotations
+
+from typing import Optional
+
+from fastapi import APIRouter, FastAPI
+
+from agno.agent.agent import Agent
+from agno.app.copilotkit.router import get_router as get_copilotkit_router
+from agno.app.fastapi.app import FastAPIApp
+from agno.app.settings import APIAppSettings
+from agno.team.team import Team
+
+
+class CopilotKitApp(FastAPIApp):
```

Comment:
```suggestion
class AGUIApp(FastAPIApp):
```

Reply:
Actually this shouldn't inherit from the FastAPIApp. We'll clean it up in another PR. For now just re-implement everything here